examples/app/stock_applet/stock_app.py

```diff
@@ -198,10 +217,13 @@ def input_change(self, obj, attrname, old, new):
             self.ticker2 = new
         if obj == self.ticker1_select:
             self.ticker1 = new
-        self.make_source()
-        self.make_plots()
-        self.set_children()
-        curdoc().add(self)
+
+        # avoid trying to compare same stocks
+        if self.ticker2 != self.ticker1:
```

Comment:
Not from python, but you can make use of the bootstrap modal in the bokehjs side